cmake/Modules/drishti_set_hunter_gate.cmake

```diff
@@ -1,4 +1,4 @@
 macro(drishti_set_hunter_version DRISHTI_HUNTER_GATE_URL DRISHTI_HUNTER_GATE_SHA1)
-  set(${DRISHTI_HUNTER_GATE_URL} "https://github.com/ruslo/hunter/archive/v0.19.116.tar.gz")
-  set(${DRISHTI_HUNTER_GATE_SHA1} "c872605eee88941a751fb91a028cc5a69c77eac0")
+  set(${DRISHTI_HUNTER_GATE_URL} "https://github.com/ruslo/hunter/archive/v0.19.123.tar.gz")
+  set(${DRISHTI_HUNTER_GATE_SHA1} "57d07480686f82ddc916a5980b4f2a18e5954c2b")
 endmacro()
```

Comment:
It would be nice to figure out a way to automatically link the Hunter release to the shield.io badge: https://img.shields.io/badge/hunter-v0.19.94-blue.svg

Reply:
What is the purpose/use case? Currently I've added badge to "available since" (example: https://docs.hunter.sh/en/latest/packages/pkg/Catch.html) but the usefulness is quite questionable for me.

Reply:
> What is the purpose/use case? 

It would be helpful to find a github project, for example, and see that it can be used through Hunter.  

> Currently I've added badge to "available since" 

I agree, "available since" is a more meaningful tag than the release # used internally.  A simple static "hunter" badge (similar to a license badge) is probably adequate.

Reply:
> It would be helpful to find a github project, for example, and see that it can be used through Hunter

I guess it will be enough to provide link to `pkg.*` page, e.g.: [![Boost](https://img.shields.io/badge/hunter-pkg.boost-blue.svg)](https://docs.hunter.sh/en/latest/packages/pkg/Boost.html) ?